legacy/firmware/protob/Makefile

```diff
@@ -2,7 +2,7 @@ ifneq ($(V),1)
 Q := @
 endif
 
-SKIPPED_MESSAGES := Binance Cardano DebugMonero Eos Monero Ontology Ripple SdProtect Tezos WebAuthn \
+SKIPPED_MESSAGES := ButtonRequest Binance Cardano DebugMonero Eos Monero Ontology Ripple SdProtect Tezos WebAuthn \
```

Comment:
errrm no i don't think this is a good idea.

Reply:
https://github.com/trezor/trezor-firmware/pull/3998/commits/88a99d7b3837c5e7d7074cd0a9fc21b85f5bff40#diff-a51b41237746944e572daaac1fc4a2f6d4c3b8e1e8db2aadb94c38e226f086a9R5